library/prolog_breakpoints.pl

```diff
@@ -61,18 +61,25 @@
 */
 
 %!  set_breakpoint(+File, +Line, +Char, -Id) is det.
+%!  set_breakpoint(+File, +Line, +Char, :Cond, -Id) is det.
 %!  set_breakpoint(+Owner, +File, +Line, +Char, -Id) is det.
 %
 %   Put a breakpoint at the indicated source-location. File is a current
 %   sourcefile (as reported by source_file/1). Line  is the 1-based line
 %   in which Char is. Char is the position of the break.
 %
+%   Cond is a goal that will be invoked whenever the newly set
+%   breakpoint is triggered. If goal fails, the breakpoint is skipped
+%   and execution commences normaly.  In the context of Cond,
```

Comment:
normaly => normally

Reply:
Thanks, fixed in https://github.com/SWI-Prolog/swipl-devel/pull/991/commits/03a3e5fb542b44d37d583be72c10c1c9736adc90